src/query/config/src/outer_v0.rs

```diff
@@ -1306,6 +1336,11 @@ pub struct MetaConfig {
 
     #[clap(long = "meta-rpc-tls-meta-service-domain-name", default_value_t)]
     pub rpc_tls_meta_service_domain_name: String,
+
+    /// MetaStore type: remote or embedded
+    #[clap(long = "meta-type", default_value = MetaType::Remote.to_str())]
+    #[serde(alias = "meta_type")]
```

Comment:
I prefer `type`.

Reply:
all field in MetaConfig named prefix by `meta-`